<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QrCodeConnectorPage</class>
 <widget class="QWizardPage" name="QrCodeConnectorPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="title">
   <string>QR-Code Transfer</string>
  </property>
  <property name="subTitle">
   <string>Paste the QR-Code data that you obtained from the partner here. Visit https://kpt.skycoder42.de to obtain the data.</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="1">
    <widget class="QToolButton" name="pasteButton">
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonFollowStyle</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="undoRedoEnabled">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Paste the QR-Code data here to continue</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionPaste">
   <property name="icon">
    <iconset theme="edit-paste" resource="../../webapp/webapp.qrc">
     <normaloff>:/icons/copy.svg</normaloff>:/icons/copy.svg</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste Qr-Code-Data from the clipboard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+V</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../webapp/webapp.qrc"/>
 </resources>
 <connections/>
</ui>
